playbooks/common/openshift-cluster/upgrades/pre/verify_control_plane_running.yml

```diff
@@ -9,16 +9,29 @@
       local_facts:
         ha: "{{ groups.oo_masters_to_config | length > 1 }}"
 
-  - name: Ensure HA Master is running
-    service:
-      name: "{{ openshift.common.service_type }}-master-api"
-      state: started
-      enabled: yes
-    when: openshift.common.is_containerized | bool
+  - when: openshift.common.is_containerized | bool
```

Comment:
I wonder why it was only checking this for containerized installs?

Reply:
Don't have answer for that. Maybe, the ha deployment was primarily for the containerized deployments? The rpm based ha deployment was never thoroughly tested?

Reply:
Maybe @dgoodwin has the answer? :)

Reply:
Afraid I do not know, looks like it goes all the way back to a7efbbb35c77213d9805ffc7354e3bd31078f38b

Reply:
@dgoodwin thanks. @sdodson this is not a blocker for the PR. Let's go forward and merge this PR. We can revisit the matter once all the bugs are resolved and see if it make sense to extend the check on rpm based deployment.